internal/apigw/apigw_openapi.go

```diff
@@ -83,6 +84,25 @@ func (module *Module) storeCanonicalRoute(route string, tokens []apigw_v1.RouteT
 	return routeData
 }
 
+func (module *Module) operationSummary(operation *apigw_v1.Operation, method pgs.Method) string {
+	if operation.Summary != "" {
+		return operation.Summary
+	}
+
+	ret := method.Name().Transform(caser.String, caser.String, " ").String()
+	ret = strings.Join(camelcase.Split(ret), " ")
+	ret = space.ReplaceAllString(ret, " ")
+	return ret
+}
```

Comment:
I think `protoc-gen-star` has a `.Name(stringVar).UpperTitleCase()` function, but this works too.

Reply:
I don't think it does actually. And specifically here I want to do it twice so that `GetAppEntitlement` turns into `Get App Entitlement`.